installers/toradex-apalis/make-install-disk.sh

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env bash
+
```

Comment:
Add check for root to address #5 
```suggestion
# Ensure root permissions to mount/unmount disks
if [ "$EUID" -ne 0 ]
  then echo "Please run as root"
  exit
fi
```
https://stackoverflow.com/a/18216122

Reply:
Thanks. Used 

```bash
 if ! [ $(id -u) = 0 ]; then                                                      
     echo >&2 "This script requires root. Please run as root."                     
     exit 2                                                                        
  fi 
```

instead because it is less likely to fail on a weird system. and execution speed isn't a concern here.